### DISCUSSION TITLE
Configuration array: how to hook in before each Sass compilation

### DISCUSSION THREAD

**Discussion by Cristialt** [webpack/webpack#16075] [Q&A]
Hi 

I have an array of configs for a number of sites sharing same design structures
On build action I'd like to enter the process just before the Sass compiler is running to be able to change a file content.

Using sass-loader options or webpack shell plugin before/after didn't help at all

webpack 4.x

Any help would be much appreciated!

Thanks,

**Comment by vankop** [comment]
> to be able to change a file content.

you should write a loader ad put it before `sass-loader`
